test/crypto-service/test_accountUpdateTransaction.js

```diff
@@ -0,0 +1,1041 @@
+    });
+  });
+
+  describe("Staked ID", async function () {
+    async function verifyAccountStakedAccountIdUpdate(stakedAccountId) {
+      // If the account was updated successfully, the queried account's staked account IDs should be equal.
+      expect(memo).to.equal(await consensusInfoClient.getAccountInfo(accountId).stakedAccountId);
+      expect(memo).to.equal(await mirrorNodeClient.getAccountData(accountId).accounts[0].staked_account_id);
+    }
+
+    async function verifyAccountStakedNodeIdUpdate(stakedAccountId) {
+      // If the account was updated successfully, the queried account's staked node IDs should be equal.
+      expect(memo).to.equal(await consensusInfoClient.getAccountInfo(accountId).stakedNodeId);
+      expect(memo).to.equal(await mirrorNodeClient.getAccountData(accountId).accounts[0].staked_node_id);
+    }
+
+    it ("(#1) Updates the staked account ID of an account to the operator's account ID", async function () {
+      // Attempt to update the staked account ID of the account to the operator's account ID.
+      const stakedAccountId = process.env.OPERATOR_ACCOUNT_ID;
+      const response = await JSONRPCRequest("updateAccount", {
+        accountId: accountId,
+        stakedAccountId: stakedAccountId,
+        commonTransactionParams: {
+          signers: [
+            accountPrivateKey
+          ]
+        }
+      });
+      if (response.status === "NOT_IMPLEMENTED") this.skip();
+
+      // Verify the staked account ID of the account was updated. 
+      verifyAccountStakedAccountIdUpdate(stakedAccountId);
+    });
+
+    it ("(#2) Updates the staked node ID of an account to a valid node ID", async function () {
+      // Attempt to update the staked node ID of the account to a valid node ID.
+      const stakedNodeId = 0;
+      const response = await JSONRPCRequest("updateAccount", {
+        accountId: accountId,
+        stakedNodeId: stakedNodeId,
+        commonTransactionParams: {
+          signers: [
+            accountPrivateKey
+          ]
+        }
+      });
+      if (response.status === "NOT_IMPLEMENTED") this.skip();
+
+      // Verify the staked node ID of the account was updated. 
+      verifyAccountStakedNodeIdUpdate(stakedNodeId);
+    });
+
+    it ("(#3) Updates the staked account ID of an account to an account ID that doesn't exist", async function () {
+      try {
+        // Attempt to update the staked account ID of the account to an account ID that doesn't exist. The network should respond with an INVALID_STAKING_ID status.
+        const response = await JSONRPCRequest("updateAccount", {
+          accountId: accountId,
+          stakedAccountId: "123.456.789",
+          commonTransactionParams: {
+            signers: [
+              accountPrivateKey
+            ]
+          }
+        });
+        if (response.status === "NOT_IMPLEMENTED") this.skip();
+      } catch (err) {
+        assert.equal(err.data.status, "INVALID_STAKING_ID");
```

Comment:
Currently fails with `FAIL_INVALID` should be investigated on services side.

Reply:
C++ is seeing `INVALID_STAKING_ID` here. What version of services are you testing with?

Reply:
I am using local-node with the following config:
```json
    {"key": "NETWORK_NODE_IMAGE_TAG", "value": "0.52.0"},
    {"key": "HAVEGED_IMAGE_TAG", "value": "0.52.0-alpha.6"},
```

This is the latest version of the local-node config.

Reply:
My bad, actually `#6` was failing, not `#3`